tests/WordMatcherTest.php

```diff
@@ -25,15 +25,25 @@
  */
 final class WordMatcherTest extends TestCase
 {
+
     /**
      * @dataProvider provideMatchCases
+     * 
+     * @param string $expected
```

Comment:
- all the `@param` / `@return` annotations are already self-documented in code itself, no need to repeat that information
- information in annotation is inaccurate (`string` !== `?string`)
- we follow same ruleset globally, if we would want to add such docs, why only in this single place?

for all the above, rejecting the proposal